.env.development

```diff
@@ -7,7 +7,7 @@
 # feature flags
 VITE_FEATURE_THORCHAIN_TCY_ACTIVITY=true
 VITE_FEATURE_QUICK_BUY=true
-VITE_FEATURE_NEW_WALLET_MANAGER=true
+VITE_FEATURE_NEW_WALLET_MANAGER=false
```

Comment:
Reverted for testing - this doesn't play nicely with new wallet popover and we may want to tweak a thing or two there. 
Not introduced by this PR per se, but made obvious by it for Ledger since we now handle disconnections, see Ledger disconnected with flag on:

<img width="1426" height="966" alt="image" src="https://github.com/user-attachments/assets/3d22febc-6798-408b-a1f3-680bdc75d2ad" />

Clicking this guy opens the connect wallet modal, but should allow you to "properly disconnect" or switch wallet provider i.e with flag off:

<img width="348" height="420" alt="image" src="https://github.com/user-attachments/assets/183aea4e-3bc2-41f2-9854-b6360c78dcd7" />

Authored for it